nyc.config.json

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  'check-coverage': true,
+  lines: 100,
+  statements: 100,
+  functions: 100,
+  branches: 100,
+  watermarks: {
```

Comment:
(suggestion) 80?

Reply:
Set same as before, just renamed file. Not sure why to drop existing rate if it's good? And package is tiny.